control/proto/gateway.proto

```diff
@@ -10,22 +10,10 @@
 
 syntax = "proto3";
 
-enum TransportType {
-    INVALID = 0;
-    RDMA = 1;
-    FC = 2;
-    TCP = 3;
-    PCIE = 256;
-    VFIOUSER = 1024;
-    CUSTOM = 4096;
-}
-
 enum AddressFamily {
     invalid = 0;
     ipv4 = 1;
     ipv6 = 2;
-    ib = 3;
```

Comment:
1. Could we make it "IPv4" and "IPv6", matching the SPDK [json RPC style](https://spdk.io/doc/jsonrpc.html), see listen_address. Maybe we could avoid doing "toupper" "tolower" this way.
2.  Could we remove "invalid" as an option, making IPv4 = 0, and IPv6 = 1?

Reply:
@baum I prefer to use the upper and lower as I don't want to rely on the different representations in different components. And, I made sure to make all the first values invalid to make sure we don't use a default value implicitly without meaning to. This way, if we don't set the value explicitly we'll get an error.